src/docfx/build/page/BookmarkValidator.cs

```diff
@@ -45,7 +46,12 @@ public List<(Error error, Document file)> Validate()
                     continue;
                 }
 
-                result.Add((Errors.BookmarkNotFound(file, reference, bookmark), file));
+                var suggestedBookmark = bookmarks != null ?
```

Comment:
There should be a threshold here, like <= 3 edits. 

Consider have a `string Levenshtein.GetMinDistance(string value, IEnumerable<string> candicates, int threshold = 3)` method.